choccake-react/src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+export default class Header extends Component {
+  render(){
+    return (
+      <>
+        <h1 className = 'header'> Chocolate Pizza</h1>
+        < img className = 'image' src = "fb-icon.png" alt ='Facebook'/> 
```

Comment:
lines 8-15 should be refactored into a ButtonList component that you call inside this header